backend/utils/db.py

```diff
@@ -1,8 +1,9 @@
 import requests
 
 PLUGIN_ID = "617db02f27c36150b422bc4d"
-ORG_ID = "61695d8bb2cc8a9af4833d46"
+ORG_ID = "619ba4671a5f54782939d384"
 COLLECTION_NAME = "rooms"

```

Comment:
We should rename and move it to `room_utils.py`
```suggestion
ROOM_COLLECTION = "rooms"

```

Reply:
fixed

Reply:
chat_rooms. Ok